### PR TITLE
Allow overriding the persistence directory

### PR DESCRIPTION
**Pull request by mmerkes**

### CODE REVIEW

File: cmd/stop.go [hunk 1]
```diff
@@ -26,6 +26,8 @@ var stopCmd = &cobra.Command{
 	Short: "Stop a task",
 	Long:  `Stops working on a task and updates the end time.`,
 	Run: func(cmd *cobra.Command, args []string) {
```
Comment (MyannaHarris) [inline cmd/stop.go:28]: This question is a little late. But where does this function get the command and strings from?

File: cmd/stop.go [hunk 2]
```diff
@@ -26,6 +26,8 @@ var stopCmd = &cobra.Command{
...
+		// Try to set the root persistence directory.
+		clerk.SetRootPersistenceDir(rootPersistenceDir)
```
Comment (MyannaHarris) [inline cmd/stop.go:30]: Does cobra automatically check if rootPersistenceDir is not set and do nothing here if it is?

File: cmd/stop.go (hunk 1 above)
Comment (mmerkes) [reply cmd/stop.go:28]: What do you mean?

File: cmd/stop.go (hunk 2 above)
Comment (mmerkes) [reply cmd/stop.go:30]: It just sets it to an empty string. This is an override, so it does nothing if the string is empty.

**Review by MyannaHarris — APPROVED** [review]

**Pull request merged**